front/app/translations/admin/en.json

```
@@ -889,6 +889,14 @@
   "app.containers.Admin.inspirationHub.SortAndReset.start_at_asc": "Start date (oldest first)",
   "app.containers.Admin.inspirationHub.SortAndReset.start_at_desc": "Start date (newest first)",
   "app.containers.Admin.inspirationHub.Survey": "Survey",
+  "app.containers.Admin.inspirationHub.UpsellNudge.bulletPoint1": "Curated list of the best projects around the globe.",
+  "app.containers.Admin.inspirationHub.UpsellNudge.bulletPoint2": "Talk & learn from fellow practitioners.",
```

```suggestion
  "app.containers.Admin.inspirationHub.UpsellNudge.bulletPoint2": "Talk to, and learn from fellow practitioners.",
```

@luucvanderzee Something is up with these translations, as Jelena pointed out this grammar error, so it seems somehow these translations have already entered the proofreading flow. 🤷